init-tools.ps1

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Load Branch Info
+cat "$RepoRoot\branchinfo.txt" | ForEach-Object {
+    if(!$_.StartsWith("#") -and ![String]::IsNullOrWhiteSpace($_)) {
+        $splat = $_.Split([char[]]@("="), 2)
+        Set-Content "env:\$($splat[0])" -Value $splat[1]
+    }
+}
+
+# Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
+if (!$env:DOTNET_INSTALL_DIR)
+{
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\$Architecture"
```

Comment:
Where is $Architecture declared?  it seems like it should be a parameter into this script file.